app/assets/stylesheets/content/_pagination.sass

```diff
@@ -97,7 +97,7 @@ $pagination--font-size: 0.8125rem
 
 .pagination--label
   +flex(1)
-  padding: 3px 0
+  padding: 3px 5px 3px 0
```

Comment:
> This also provides a minimal right padding for the pagination--label, as its otherwise too close to the per_page options (cf. screenshot)

Isn't the thing to increase for better spacing between one thing and another the `margin`?

It makes no visual difference at the moment, because the label has neither a border nor a background color, but I believe semantically we should be using a margin for that purpose.

And that is basically the only thing I know about CSS :wink: